enterprise-suite/scripts/lbc.py

```diff
@@ -288,16 +290,54 @@ def are_cluster_roles_created():
         fail_msg='Found some cluster roles from previous console install, but not all: {}. Please clean them up manually.'
     )
 
+# Takes helm style "key1=value1,key2=value2" string and returns a list of (key, value)
```

Comment:
markus, did you consider passing command line arg as is to helm.....

Reply:
It already has that feature.  Anything after an arg of `--` gets passed on to helm.  (You might find the discussion in https://github.com/lightbend/es-backend/issues/474 of interest.)